bundles/admin/admin-layereditor/view/AdminLayerFormService.js

```diff
@@ -119,28 +174,29 @@ export class AdminLayerFormService {
         }
 
         me.layer = {
-            version: layer ? layer.getVersion() : '',
-            layer_id: layer ? layer.getId() : null,
-            layerUrl: layer ? layer.getAdmin().url : '',
-            username: layer ? layer.getAdmin().username : '',
-            password: layer ? layer.getAdmin().password : '',
-            layerName: layer ? layer.getLayerName() : '',
+            type: layer.getLayerType(),
+            version: layer.getVersion(),
+            layer_id: layer.getId(),
+            url: layer.getAdmin().url,
+            username: layer.getAdmin().username,
+            password: layer.getAdmin().password,
+            layerName: layer.getLayerName(),
             ...this._getLocalizedLayerInfo(layer),
-            groupId: layer ? layer.getAdmin().organizationId : null,
-            organizationName: layer ? layer.getOrganizationName() : '',
-            maplayerGroups: layer ? [...layer.getGroups()] : [],
-            opacity: layer ? layer.getOpacity() : 100,
-            minScale: layer ? layer.getMinScale() : 1,
-            maxScale: layer ? layer.getMaxScale() : 1,
-            style: layer ? layer.getCurrentStyle().getName() : '',
-            styleTitle: layer ? layer.getCurrentStyle().getTitle() : '',
+            groupId: layer.getAdmin().organizationId,
+            organizationName: layer.getOrganizationName(),
+            maplayerGroups: [...layer.getGroups()],
+            opacity: layer.getOpacity() || 100,
+            minScale: layer.getMinScale() || 1,
```

Comment:
The scale defaults still need some work, but it's not the focus here, so this is ok.